tests/dcosjob.py

```diff
@@ -0,0 +1,37 @@
+import os
```

Comment:
Ideally, this file wouldn't exist, and we would be calling into `shakedown` or `dcos` for all these functions.